workers/unity/Assets/Playground/Scripts/Worker/ConnectGameLogicWorker.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+using System.Threading.Tasks;
+using Improbable.Gdk.Core;
+using Playground;
+using Unity.Entities;
+using UnityEngine;
+
+public class ConnectGameLogicWorker : MonoBehaviour
+{
+    public Vector3 Origin = Vector3.zero;
+    public GameObject Level;
+    public int ConnectionAttempts = 3;
```

Comment:
Nit ConnectionAttempts -> MaxConnectionAttempts 
(Given it's not a count of the number of elapsed connection attempts)